packages/main/src/system/macos-startup.spec.ts

```diff
@@ -66,35 +66,15 @@ beforeEach(() => {
   // fake path.resolve by just adding /
   vi.mocked(resolve).mockImplementation((...args: string[]) => args.join('/'));
 
-  vi.mocked(app.getPath).mockImplementation(
-    (
-      name:
-        | 'home'
-        | 'appData'
-        | 'userData'
-        | 'sessionData'
-        | 'temp'
-        | 'exe'
-        | 'module'
-        | 'desktop'
-        | 'documents'
-        | 'downloads'
-        | 'music'
-        | 'pictures'
-        | 'videos'
-        | 'recent'
-        | 'logs'
-        | 'crashDumps',
-    ) => {
-      if (name === 'exe') {
-        return fakeAppExe;
-      }
-      if (name === 'home') {
-        return fakeAppHome;
-      }
-      throw new Error('Unsupported path');
-    },
-  );
+  vi.mocked(app.getPath).mockImplementation((name: Parameters<typeof app.getPath>[0]) => {
+    if (name === 'exe') {
+      return fakeAppExe;
+    }
+    if (name === 'home') {
+      return fakeAppHome;
+    }
+    throw new Error('Unsupported path');
```

Comment:
it is not related to the referenced changes
it should probably be a quick refactor PR targeting the replacement of the args by typeof app.getPath

issue is for Windows, this file is for macOS, etc.